commands/docker-compose.ts

```diff
@@ -55,20 +55,21 @@ async function compose(commands: ('up' | 'down')[], message: string, dockerCompo
 
     let selectedItems: Item[] = [];
```

Comment:
simplify this (you've got conditions for essentially the same thing in two different places):

let selectedItems = (selectedComposeFileUris && selectedComposeFileUris.length) ? selectedComposeFileUris : [dockerComposeFileUri];
if (!selectedItems.length) {
  // prompt for compose file

Reply:
given that the first line still stores an array of URI's , what do you feel about assigning it back to the same variable `selectedComposeFileUris `?

Reply:
Not going to do the above. My bad.